product_docs/docs/bdr/3.7/repsets.mdx

```diff
@@ -278,13 +278,11 @@ transaction.
     another node, because this will stop replication on that
     node. Should this happen, please unsubscribe the affected node
     from that replication set.  
-
-    For the same reason, you should not drop a replication set if
+     the same reason, you should not drop a replication set if
     there is a join operation in progress, and the node being joined
     is a member of that replication set; replication set membership is
     only checked at the beginning of the join.  
-
-    This happens because the information on replication set usage is
+    s happens because the information on replication set usage is
```

Comment:
Looks like someone d a word...

Reply:
Oddly, I don't see that in the source material: https://github.com/EnterpriseDB/bdr/blob/548b3d63118861bfbbee0701a144f8457f8978b2/docs/docs2/repsets.md?plain=1#L282

Reply:
Weird; this looks like a bug in the admonitions component: I see now that it did the same thing on line 281 - both are indented with a tab (vs 4 spaces); betting it's assuming there will be a four-space indentation and just eating the first four characters.